src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -394,8 +394,14 @@ <h2>
                 </div>
             </ng-container>
         </div>
-        <jhi-assessment-instructions class="max-vh-100" style="max-height: 100vh" *ngIf="exercise.type === ExerciseType_MODELING" [exercise]="exercise" [collapsed]="true">
-        </jhi-assessment-instructions>
+        <jhi-collapsable-assessment-instructions
+            *ngIf="[ExerciseType.MODELING, ExerciseType.TEXT].includes(exercise.type)"
+            class="max-vh-100"
+            style="max-height: 100vh"
```

Comment:
Are both the class and the style attribute needed?

Reply:
I found out that the class does not even exists. I moved the style to the component styles.

Reply:
Oh, you are right. Looks like Bootstrap 4 has utility classes for `max-width` and `min-viewport-width` but not for `min-width` and `max-viewport-width` ([code](https://github.com/twbs/bootstrap/blob/master/scss/_utilities.scss#L104)). Do they expect people to memorize that? :smile: